components/merino/src/curated_recommendations/error.rs

```diff
@@ -55,6 +55,7 @@ impl GetErrorHandling for Error {
                     code: Some(*code),
                     reason: format!("Validation error: {}", message),
                 })
+                .report_error("merino-validation")
```

Comment:
I wasn't sure if this is all that's required to report to sentry?

Reply:
Yup that's it.